mfr/providers/osf/provider.py

```diff
@@ -96,6 +99,12 @@ def __init__(self, request, url):
         # }}
 
         name, ext = os.path.splitext(metadata['data']['name'])
+
+        if MAX_FILE_SIZE_TO_RENDER.get(ext) and metadata['data']['size'] > MAX_FILE_SIZE_TO_RENDER.get(ext):
+            size = sizeof_fmt(MAX_FILE_SIZE_TO_RENDER.get(ext))
+            raise TooBigToRenderError("This file with extension '{ext}' exceeds the size limit of {size} and will not "
```

Comment:
Max line length for MFR should be 100 chars, same as WaterButler.  I thought this was in the CONTRIBUTING, but I'm not seeing it.  Boo on me.